packages/s/soloud/xmake.lua

```diff
@@ -0,0 +1,52 @@
+package("soloud")
+    set_description("SoLoud is an easy to use, free, portable c/c++ audio engine for games.")
+    
+    set_homepage("https://sol.gfxile.net/soloud/")
+    set_license("zlib")
+    
+    add_urls("https://github.com/jarikomppa/soloud/archive/refs/tags/RELEASE_20200207.zip")
```

Comment:
```
add_urls("https://github.com/jarikomppa/soloud/archive/refs/tags/RELEASE_$(version).zip", function (version)
        return version:gsub("%.", "")
    end)
```

Reply:
I get an error with this: `error: @programdir/core/main.lua:300: @programdir/modules/devel/git/checkurl.lua:28: attempt to index a function value (local 'url')`

Reply:
I fixed it.

Reply:
```
add_urls("https://github.com/jarikomppa/soloud/archive/refs/tags/RELEASE_$(version).zip", {version = function (version)
    return version:gsub("%.", "")
end})
```